Modules/Payments/Controllers/PaymentMethodsController.php

```
@@ -58,39 +50,37 @@ public function index(int $page = 0): \Illuminate\View\View
      * @return \Illuminate\View\View|\Illuminate\Http\RedirectResponse
      *
      * @legacy-function form
-     *
      * @legacy-file application/modules/payment_methods/controllers/Payment_methods.php
-     *
-     * @legacy-line 42
      */
-    public function form($id = null)
+    public function form(?int $id = null)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)